pkg/controller/controller.go

```diff
@@ -777,7 +777,7 @@ func (c *controller) reconcileInstance(instance *v1alpha1.Instance) error {
 		if respCode == http.StatusAccepted {
 			glog.V(5).Infof("Received asynchronous provisioning response for Instance %v/%v of ServiceClass %v at Broker %v: response: %v", instance.Namespace, instance.Name, serviceClass.Name, brokerName, response)
 			if response.Operation != "" {
-				instance.Status.LastOperation = response.Operation
+				instance.Status.LastOperation = &response.Operation
```

Comment:
not 100% sure its the same but check this out:  https://play.golang.org/p/yTIm0ig2JZ